### DISCUSSION TITLE
Brushes

### DISCUSSION THREAD

**Discussion by AlexSSSx** [keiwando/toonsquid#182] [General]
What do you think about brushes? There are a little bit primitive in toonsquid. Brushes with dynamic flow in procreate, photoshop, even in callipeg, can do magic things. But the same brushes with the same settings in toonsquid are absolutely useless. Do anybody have any plans to improve toonsquid brushes?
However toonsquid — awesome animation app. Extremely like it. Very simple but full of useful features.

**Comment by AlexSSSx** [comment]
![IMG_1963](https://user-images.githubusercontent.com/127799650/224851151-68b95d1e-81d7-407f-956e-c0a1e11d939c.jpeg)

**Comment by DinoAwakening** [comment]
Primitive you say? I think you didn’t even try to use all the stuff inside the brush settings, ngl… :/

**Reply by AlexSSSx** [reply]
I tried some. But may be I do something wrong. Could you show me please what stuff should I set for getting the result as I have with procreate on my example picture? Or may be you can send me already well-tuned brush?

**Comment by Cat-Butler** [comment]
I’m probably mistaken, but it looks like in the other apps you've used a soft-edged brush with variable flow, while in ToonSquid you're using a hard-edged brush with variable opacity, such as the Dynamic Opacity brush in the Painting set. If so, you would be better off starting with a brush from the Airbrush set instead.

For example, here's a quick experiment using a brush with minor changes from the Soft Airbrush, with only the Flow changed by Pencil pressure, not Opacity. I did change the size and opacity manually, but I didn't spend too much time on the brush, it could have been tuned further.

https://youtu.be/aUBJImtuEbU

Here's a copy of the brush, but it isn't anything special.

[SoftShader.zip](https://github.com/keiwando/toonsquid/files/11010726/SoftShader.zip)

**Reply by AlexSSSx** [reply]
I tried hard-edged brush with variable flow for all 3 examples. You are right — soft edged brush gives better result almost the same I looking for. But not exactly I need. My goal is exact a hard-edged brush with variable flow, which can make not only half-clay-half-plastic effect on the render (if you know what I mean). 
Somebody from toonsquid dev team answer for my mail and suggested that the reason of my issue can be 8-bit colors whereas Procreate uses 16-bit colors. And they planning on adding 16bit color support at some point in the future.
